WASM.md

```diff
@@ -0,0 +1,36 @@
+Support for WebAssembly
+=======================
+
+# Summary
+
+Due to the need for a standard library in the build these quantum routines it is not
+possible to compile them as pure no-std WASM however by using wasi and its standard
+library it becomes possible.
+
+# Steps
+
+Download the wasm32-wasi sysroot build from https://github.com/WebAssembly/wasi-sdk
+and set the environment variable to point to it
+
+```sh
+cd ..
+wget https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-12/wasi-sysroot-12.0.tar.gz
+tar -xvzf wasi-sysroot-12.0.tar.gz
+rm wasi-sysroot-12.0.tar.gz
+
+sudo tee /etc/profile.d/wasi.sh <<EOF
+export WASI_SDK_DIR="$(pwd)/wasi-sysroot"
+EOF
+source /etc/profile.d/wasi.sh
+````
```

Comment:
This has one backtick too many (I can't make a suggestion here, Github breaks on it)

Reply:
fixed